### ISSUE TITLE
Dawg wut

### ISSUE THREAD

**Issue by LegitGremlin** [fatrolls/IconChanger#1]
Nice opinion, one small issue: https://www.virustotal.com/gui/file/54d6bd8495e9b3fb3348e684b98e3e9de01643008afe8cddd541546e114c32da

**Comment by fatrolls** [comment]
Yeah those are bad virus scanners they detect it as a virus they are all just false positives if you pack the program in a packer like VMProtect/Themida then the virus warnings will go away
